src/Physics/QuasiElastic/XSection/ZExpELFormFactorModel.cxx

```diff
@@ -0,0 +1,736 @@
+  + llb2*( -3.*(3*fKmax+4) + llp4*llp2*(2*fKmax+3)*zllp1        \
+         -3.*llp4*llp1*llp1*zllp2 + llp2*llp1*llp0*zllp4        )  \
+  + llb1*(  3.*llp1*(3*fKmax+8) - llp4*llp3*llp2*llp1*zllp1        \
+  +(3./2.)*llp4*llp3*llp1*llp0*zllp2 - .5*llp2*llp1*llp1*llp0*zllp4) );
+
+  fZ_BNn[(int)llp2] = (1./lldenom) *                           \
+  ( 3.*(llb0-llb0z)*llp4*llp3*llp1                                \
+  + llb3*( -3. + .5*llp4*llp3*zllp1 - (3./2.)*llp4*llp1*zllp3      \
+         +llp3*llp1*zllp4                                  )  \
+  + llb2*(  3.*(3*fKmax+5) - llp4*llp3*llp2*zllp1                \
+         +3.*llp4*llp1*llp1*zllp3 - llp3*llp1*(2*fKmax+1)*zllp4)  \
+  + llb1*( -3.*llp3*(3*fKmax+4) + .5*llp4*llp3*llp3*llp2*zllp1     \
+    -(3./2.)*llp4*llp3*llp1*llp0*zllp3 + llp3*llp2*llp1*llp0*zllp4)  );
+
+  fZ_BNn[(int)llp1] = (1./lldenom) *                           \
+  ( -(llb0-llb0z)*llp4*llp3*llp2                                  \
+  + llb3*(  1. - .5*llp4*llp3*zllp2 + llp4*llp2*zllp3              \
+         -.5*llp3*llp2*zllp4                               )  \
+  + llb2*( -3.*llp2 + llp4*llp3*llp2*zllp2                        \
+         -llp4*llp2*(2*fKmax+3)*zllp3 + llp3*llp2*llp1*zllp4)     \
+  + llb1*(  3.*llp3*llp2 - .5*llp4*llp3*llp3*llp2*zllp2             \
+         +llp4*llp3*llp2*llp1*zllp3 - .5*llp3*llp2*llp2*llp1*zllp4)  );
+
+  fZ_BNn[0] = (1./lldenom) *                                  \
+  ( -6.*llb0z                                                \
+  + llb0*(  llp4*llp3*llp2*zllp1 - 3.*llp4*llp3*llp1*zllp2           \
+         +3.*llp4*llp2*llp1*zllp3 - llp3*llp2*llp1*zllp4        )  \
+  + llb3*( -zllp1 + 3.*zllp2 - 3.*zllp3 + zllp4               )  \
+  + llb2*(  3.*llp2*zllp1 - 3.*(3*fKmax+5)*zllp2                \
+         +3.*(3*fKmax+4)*zllp3 - 3.*llp1*zllp4             )  \
+  + llb1*( -3.*llp3*llp2*zllp1 + 3.*llp3*(3*fKmax+4)*zllp2        \
+         -3.*llp1*(3*fKmax+8)*zllp3 + 3.*llp2*llp1*zllp4     )  );
+
+
+
+
+}
+//____________________________________________________________________________
+void ZExpELFormFactorModel::Configure(const Registry & config)
+{
+  Algorithm::Configure(config);
+  this->LoadConfig();
+}
+//____________________________________________________________________________
+void ZExpELFormFactorModel::Configure(string param_set)
+{
+  Algorithm::Configure(param_set);
+  this->LoadConfig();
+}
+//____________________________________________________________________________
+void ZExpELFormFactorModel::LoadConfig(void )
+{
+// get config options from the configuration registry or set defaults
+// from the global parameter list
+
+  GetParam( "QEL-Q4limit", fQ4limit ) ;
+  GetParam( "QEL-Kmax", fKmax ) ;
+
+  GetParam( "QEL-T0", fT0 ) ;
+  GetParam( "QEL-Tcut", fTcut ) ;
+
+  GetParam( "QEL-Gep0", fGep0 ) ;
+  GetParam( "QEL-Gmp0", fGmp0 ) ;
+  GetParam( "QEL-Gen0", fGen0 ) ;
+  GetParam( "QEL-Gmn0", fGmn0 ) ;
+  assert(fKmax > 0);
+
+  // z expansion coefficients
```

Comment:
WE have ways to load vector of parameters. Example of reading is [here](https://github.com/GENIE-MC/Generator/blob/ecc6884b81eec3ef3d909a09150aa856805c3bf5/src/Physics/Decay/BaryonResonanceDecayer.cxx#L917) and an example of xml file is [here](https://github.com/GENIE-MC/Generator/blob/ecc6884b81eec3ef3d909a09150aa856805c3bf5/config/BaryonResonanceDecayer.xml#L84)